src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -82,9 +93,17 @@ private String getCloudInputs() {
                 {new BQSRTest(GRCh37Ref, hiSeqBam_1read, dbsnp_138_b37_20_21_vcf, " --joinStrategy SHUFFLE", getResourceDir() + BQSRTestData.EXPECTED_WGS_B37_CH20_1READ_RECAL)},
                 {new BQSRTest(GRCh37Ref, HiSeqBam_chr20, dbSNPb37_chr20, " --joinStrategy SHUFFLE", localResources + BQSRTestData.EXPECTED_WGS_B37_CH20_1M_1M1K_RECAL)},
 
-                // Currently disabled because BaseRecalibratorSpark can't handle more than 1 knownSites file: https://github.com/broadinstitute/gatk/issues/1085
-                // Re-enable once this is fixed.
-                //{new BQSRTest(GRCh37Ref, HiSeqBam_chr20, dbSNPb37, "-knownSites " + moreSites, localResources + "expected.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.2inputs.recal.txt")},
+                // multiple known sites with SHUFFLE
+                {new BQSRTest(GRCh37Ref_chr2021, hiSeqBam_20_21_100000, more20Sites, " --joinStrategy SHUFFLE -knownSites " + more21Sites, getResourceDir() + "expected.CEUTrio.HiSeq.WGS.b37.ch20.ch21.10m-10m100.recal.txt")},
+                {new BQSRTest(GRCh37Ref_chr2021, hiSeqCram_20_21_100000, more20Sites, " --joinStrategy SHUFFLE -knownSites " + more21Sites, getResourceDir() + "expected.CEUTrio.HiSeq.WGS.b37.ch20.ch21.10m-10m100.recal.txt")},
+
+                // multiple known sites with 2bit BROADCAST
+                {new BQSRTest(GRCh37Ref2bit_chr2021, hiSeqBam_20_21_100000, more20Sites, " -knownSites " + more21Sites, getResourceDir() + "expected.CEUTrio.HiSeq.WGS.b37.ch20.ch21.10m-10m100.recal.txt")},
+                // Can't use 2 bit reference with a CRAM file: https://github.com/broadinstitute/gatk/issues/1443
+                //{new BQSRTest(GRCh37Ref2bit_chr2021, hiSeqCram_20_21_100000, more20Sites, " -knownSites " + more21Sites, getResourceDir() + "expected.CEUTrio.HiSeq.WGS.b37.ch20.ch21.1m-1m100.recal.txt")},
+
+                // multiple known sites test case taken from the non-Spark BaseRecalibratorIntegrationTest, with SHUFFLE
+                {new BQSRTest(hg18Reference, HiSeqBam_chr17, dbSNPb37_chr17, " --joinStrategy SHUFFLE -knownSites " + moreSites, getResourceDir() + "expected.NA12878.chr17_69k_70k.2inputs.txt")},
 
                 {new BQSRTest(GRCh37Ref, HiSeqBam_chr20, dbSNPb37_chr20, " --joinStrategy SHUFFLE --indels_context_size 4",  localResources + BQSRTestData.EXPECTED_WGS_B37_CH20_1M_1M1K_INDELS_CONTEXT_SIZE_4_RECAL)},
                 {new BQSRTest(GRCh37Ref, HiSeqBam_chr20, dbSNPb37_chr20, " --joinStrategy SHUFFLE --low_quality_tail 5",     localResources + BQSRTestData.EXPECTED_WGS_B37_CH20_1M_1M1K_LOW_QUALITY_TAIL_5_RECAL)},
```

Comment:
The `DataProviders` in this class are a bit of a mess currently -- would you mind refactoring them a bit? I recommend unifying the `BQSRTest`, `BQSRLocalRefTest`, and `BQSRLocalRefTestShuffle` providers into a single provider, and grouping the test cases to make it clear to the reader which ones are tested on both BROADCAST and SHUFFLE, and with a reference file vs. the reference API.